spec/lib/metasploit/framework/login_scanner/kerberos_spec.rb

```diff
@@ -43,7 +43,8 @@
 
   let(:tgt_response_client_revoked) do
     ::Rex::Proto::Kerberos::Model::Error::KerberosError.new(
-      error_code: ::Rex::Proto::Kerberos::Model::Error::ErrorCodes::KDC_ERR_CLIENT_REVOKED
+      error_code: ::Rex::Proto::Kerberos::Model::Error::ErrorCodes::KDC_ERR_CLIENT_REVOKED,
+      res: ::Rex::Proto::Kerberos::Model::KrbError.new
```

Comment:
It'd be great to add a separate test example for the scenario of the windows-specific response being present